app/build.gradle

```diff
@@ -123,17 +123,15 @@ custom.eachFile() { file ->
     return
   }
   map.put(fileName, file.getAbsolutePath())
-
-
 }
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 25
```

Comment:
No, no issues like that, you need to target a certain version of the API to be able to use newer features though, (you disable them in older versions)